<launch>

  <node name="real_robot" pkg="robot_sim" type="real_robot.py" output="screen" />

</launch>

